docs/install-upgrade/build-wiring.md

```diff
@@ -56,14 +54,14 @@ OPTIONS:
 ```
 
 ### Sample Switch Configuration
-```yaml
+``` { .yaml .annotate linenums="1" }
 apiVersion: wiring.githedgehog.com/v1beta1
 kind: Switch
 metadata:
   name: ds3000-02
 spec:
   boot:
-    serial: ABC123XYZ
+    serial: ABC123XYZ # (1)
```

Comment:
```suggestion
    serial: ABC123XYZ # or we can also use "mac: 34:AD:61:00:02:03"
```

And get rid of the (somewhat) confusing footnote?

Reply:
By “confusing” I mostly meant that it was not super easy to find the footnote that it refers to; much of the confusion goes away if you use the same `(1)` syntax at both locations for example. I did not mean that the content of the note itself was confusing.